Makefile

```diff
@@ -59,18 +59,18 @@ CUSTOM_NPM_REGISTRY ?= $(shell git config tkg.npmregistry)
 
 # TKG Compatibility Image repo and  path related configuration
 ifndef TKG_DEFAULT_IMAGE_REPOSITORY
-TKG_DEFAULT_IMAGE_REPOSITORY = "projects-stg.registry.vmware.com/tkg"
+TKG_DEFAULT_IMAGE_REPOSITORY = "projects.registry.vmware.com/tkg"
 endif
 ifndef TKG_DEFAULT_COMPATIBILITY_IMAGE_PATH
 # TODO change it to "tkg-compatibility" once the image is pushed to registry
-TKG_DEFAULT_COMPATIBILITY_IMAGE_PATH = "framework-zshippable/tkg-compatibility"
+TKG_DEFAULT_COMPATIBILITY_IMAGE_PATH = "tkg-compatibility"
```

Comment:
gotta revert this and `pkg/v1/tkg/tkgconfigpaths/zz_bundled_default_bom_files_configdata.go` file change as those are downstream specific (being carried as patch downstream) - 

so revert https://github.com/vmware-tanzu/tanzu-framework/pull/1762/commits/9ebe78cf6f640f71ceae2dc49c71d6eba02b3571

Reply:
done